message PB_UTAGuessUserInfoData {                                //!< 玩家下注信息
    optional int32 uid = 1;                                      //!< 玩家UID
    optional int32 plat_id = 2;                                  //!< 所属平台ID
    optional int32 server_id = 3;                                //!< 所属服务器ID
    optional int64 team_seq = 4;                                 //!< 下注的队伍序号
    optional int64 match_seq = 5;                                //!< 下注的比赛序号
    optional int32 bet_coin_num = 6;                             //!< 下注的金额
    optional int32 is_get = 7;                                   //!< 是否已领取
}

message PB_UTAGuessTeamInfoData {                                //!< 竞猜队伍信息
    optional int64 team_seq = 1;                                 //!< 队伍序号
    optional int64 match_seq = 2;                                //!< 比赛序号
    optional int32 plat_id = 3;                                  //!< 所属平台ID
    optional int32 server_id = 4;                                //!< 所属服务器ID
    optional int32 total_bet_num = 5;                            //!< 被下注的总金额
    optional int32 guess_status = 6;                             //!< 竞猜状态 [0:可以下注 , 1:停止下注 , 2:开奖[获胜] , 3开奖[失败]]
}

message PB_UTAGuessMatchInfoData {                               //!< 竞猜比赛信息
    optional int64 match_seq = 1;                                //!< 比赛序号
    repeated PB_UTAGuessTeamInfoData team_info = 2;              //!< 参加该比赛的队伍信息
}

